OSRMTextInstructions.podspec

```diff
@@ -25,6 +25,10 @@ Pod::Spec.new do |s|
 
   s.source = { :git => "https://github.com/Project-OSRM/osrm-text-instructions.swift.git", :tag => "v#{s.version.to_s}" }
 
+  # ――― Resources ―――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――――― #
+
+  s.resources = ['OSRMTextInstructions/*.plist']
```

Comment:
When we start including localized versions of this plist, we’ll need to tweak the path a bit.